exoctk/utils.py

```diff
@@ -33,6 +29,36 @@
 # Set the version
 VERSION = '0.2'
 
+# Get the location of EXOCTK_DATA environvment variable and check that it is valid
+EXOCTK_DATA = os.environ.get('EXOCTK_DATA')
+
+# If the variable is blank or doesn't exist
+ON_TRAVIS = os.path.expanduser('~') == '/home/travis' or os.path.expanduser('~') == '/Users/travis'
+if not ON_TRAVIS:
+    if not EXOCTK_DATA:
+        raise ValueError(
+            'The $EXOCTK_DATA environment variable is not set.  Please set the '
+            'value of this variable to point to the location of the ExoCTK data '
```

Comment:
I've never seen a long string input to an error like this so I wanted to check in. Can you do this or are you missing some `.\`s or something?

Reply:
@julesfowler It should work since its all between parentheses!